doc/user/content/sql/create-source/mysql.md

```diff
@@ -409,6 +411,8 @@ CREATE SOURCE mz_source
 
 ### Handling errors and schema changes
 
+{{< include-md file="shared-content/schema-changes-in-progress.md" >}}
```

Comment:
https://preview.materialize.com/materialize/32436/sql/create-source/mysql/#handling-errors-and-schema-changes